Unignore test case: test_demo_dev_ops

```
1:34:48 [ERROR] - Test case failed: test_demo_dev_ops - Tests SUCCESS case - clears docker container and calls the flow
		Expected result: SUCCESS
		Actual result: EXECUTE_DEMO_DEV_OPS_PROBLEM
```

--- a/ci-env/circleci/wait_for_droplets_and_update_test_inputs.sh
+++ b/ci-env/circleci/wait_for_droplets_and_update_test_inputs.sh
@@ -118,6 +118,33 @@ do
   fi
 done
 
+# define swap space
+for DROPLET_IP in ${DROPLET_IP_ADDRESS_ACC}
+do
+  CMD_OUTPUT=$(ssh -i ${SSH_KEY_PATH} \
+  -o UserKnownHostsFile=/dev/null \
+  -o StrictHostKeyChecking=no \
+  core@${DROPLET_IP} \
+  'sudo fallocate -l 1G /swapfile && sudo chmod 600 /swapfile \
+  && sudo mkswap /swapfile && sudo swapon /swapfile \
+  && sudo sysctl vm.swappiness=10 && sudo sysctl vm.vfs_cache_pressure=50')
+
+  FREE_CMD_OUTPUT=$(ssh -i ${SSH_KEY_PATH} \
+  -o UserKnownHostsFile=/dev/null \
+  -o StrictHostKeyChecking=no \
+  core@${DROPLET_IP} \
+  'free -m')
+  SWAP_SIZE=$(echo "$FREE_CMD_OUTPUT" | grep "Swap" | awk '{print $2}')
+
+  if [ "${SWAP_SIZE}" == "1023" ]
+  then
+    echo "Swap space defined: ${CMD_OUTPUT}"
+  else
+    echo "Problem while defining swap space. Size: ${SWAP_SIZE} - OUTPUT: ${CMD_OUTPUT}"
+    exit 1
+  fi
+done
+
 # update inputs files to use actual IP addresses
 DROPLET_IP_ARRAY=(${DROPLET_IP_ADDRESS_ACC})
 COREOS_PLACEHOLDERS_ARRAY=(${COREOS_PLACEHOLDERS})

--- a/content/io/cloudslang/base/network/verify_url_is_accessible.sl
+++ b/content/io/cloudslang/base/network/verify_url_is_accessible.sl
@@ -9,7 +9,8 @@
 #!!
 #! @description: Checks if a url is accessible.
 #! @input url: the url
-#! @input time_to_sleep: time in seconds to wait
+#! @input attempts: attempts to reach host
+#! @input time_to_sleep: time in seconds to wait between attempts
 #! @input proxy_host: optional - proxy server used to access the web site
 #! @input proxy_port: optional - proxy server port
 #! @output output_message: timeout exceeded and url was not accessible
@@ -28,8 +29,9 @@ flow:
   name: verify_url_is_accessible
   inputs:
     - url
+    - attempts: 1
     - time_to_sleep:
-        default: '1'
+        default: 1
         required: false
     - content_type:
         default: "application/json"
@@ -52,7 +54,6 @@ flow:
     - proxy_port:
         required: false
   workflow:
-
     - http_get:
         do:
           rest.http_client_get:
@@ -67,19 +68,17 @@ flow:
             - keystore_password
             - proxy_host
             - proxy_port
-
         publish:
           - return_code
-
         navigate:
-          - SUCCESS: wait
+          - SUCCESS: SUCCESS
           - FAILURE: check_if_timed_out
 
     - check_if_timed_out:
          do:
             math.compare_numbers:
-              - value1: ${return_code}
-              - value2: '-1'
+              - value1: ${attempts}
+              - value2: 0
          navigate:
            - GREATER_THAN: wait
            - EQUALS: FAILURE
@@ -89,16 +88,15 @@ flow:
         do:
           utils.sleep:
               - seconds: ${time_to_sleep}
-
+              - attempts
+        publish:
+          - attempts: ${attempts - 1}
         navigate:
-          - SUCCESS: SUCCESS
+          - SUCCESS: http_get
           - FAILURE: FAILURE
-
   outputs:
     - return_code
     - output_message: ${"Url is accessible" if return_code == '0' else "Url is not accessible"}
-
-
   results:
     - SUCCESS
     - FAILURE

--- a/content/io/cloudslang/base/network/verify_url_is_accessible.sl
+++ b/content/io/cloudslang/base/network/verify_url_is_accessible.sl
@@ -10,6 +10,8 @@
 #! @description: Checks if a url is accessible.
 #! @input url: the url
 #! @input time_to_sleep: time in seconds to wait
+#! @input proxy_host: optional - proxy server used to access the web site
+#! @input proxy_port: optional - proxy server port
 #! @output output_message: timeout exceeded and url was not accessible
 #! @result SUCCESS: url is accessible
 #! @result FAILURE: url is not accessible
@@ -45,6 +47,10 @@ flow:
         default: ${get_sp('io.cloudslang.base.network.keystore_password')}
         required: false
         sensitive: true
+    - proxy_host:
+        required: false
+    - proxy_port:
+        required: false
   workflow:
 
     - http_get:
@@ -59,6 +65,8 @@ flow:
             - trust_password
             - keystore
             - keystore_password
+            - proxy_host
+            - proxy_port
 
         publish:
           - return_code

--- a/content/io/cloudslang/docker/containers/examples/demo_dev_ops.sl
+++ b/content/io/cloudslang/docker/containers/examples/demo_dev_ops.sl
@@ -26,6 +26,8 @@
 #! @input email_password: optional - email password
 #! @input email_enable_TLS: optional - enable startTLS
 #! @input timeout: optional - time in milliseconds to wait for command to complete - Default: 30000000 ms (8.33 h)
+#! @input proxy_host: optional - proxy server used to access the web site
+#! @input proxy_port: optional - proxy server port
 #!!#
 ####################################################
 namespace: io.cloudslang.docker.containers.examples
@@ -62,6 +64,10 @@ flow:
     - email_enable_TLS:
         required: false
     - timeout: '30000000'
+    - proxy_host:
+        required: false
+    - proxy_port:
+        required: false
   workflow:
 
     - create_db_container:
@@ -114,6 +120,8 @@ flow:
             - url: ${'http://' + docker_host + ':' + app_port}
             - attempts: 20
             - time_to_sleep: 10
+            - proxy_host
+            - proxy_port
         publish:
           - error_message: ${output_message}
 

--- a/test/io/cloudslang/base/network/verify_url_is_accessible.inputs.yaml
+++ b/test/io/cloudslang/base/network/verify_url_is_accessible.inputs.yaml
@@ -31,7 +31,7 @@ testVerifyUrlIsAccessible_Failure_unreachable_url:
 
 testVerifyUrlIsAccessible_Failure_Negative_TimeToSleep:
   inputs:
-    - url: 'http://www.google.com'
+    - url: 'invalid_url'
     - time_to_sleep: '-2'
   description: Tests that verify_url_is_accessible.sl operation fails with -2 time_to_sleep
   systemPropertiesFile: ${project_path}/configuration/properties/io/cloudslang/base/http/http.prop.sl

--- a/test/io/cloudslang/docker/containers/examples/demo_dev_ops.inputs.yaml
+++ b/test/io/cloudslang/docker/containers/examples/demo_dev_ops.inputs.yaml
@@ -7,7 +7,7 @@
 #
 ####################################################
 test_demo_dev_ops:
-  testSuites: [docker-ignore,docker-circleci-ignore]
+  testSuites: [docker,docker-circleci]
   description: Tests SUCCESS case - clears docker container and calls the flow
   testFlowPath: io.cloudslang.docker.containers.examples.test_demo_dev_ops
   inputs:

--- a/test/io/cloudslang/docker/containers/examples/test_demo_dev_ops.sl
+++ b/test/io/cloudslang/docker/containers/examples/test_demo_dev_ops.sl
@@ -22,6 +22,10 @@ flow:
     - email_port
     - email_sender
     - email_recipient
+    - proxy_host:
+        required: false
+    - proxy_port:
+        required: false
   workflow:
     - clear_docker_containers:
          do:
@@ -44,6 +48,8 @@ flow:
             - email_port
             - email_sender
             - email_recipient
+            - proxy_host
+            - proxy_port
         navigate:
           - SUCCESS: SUCCESS
           - FAILURE: EXECUTE_DEMO_DEV_OPS_PROBLEM

